SpywareFilter/sections/specific.txt

```diff
@@ -58,6 +58,8 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||za.zalo.me/v*/brp?fts=
 ||za.zalo.me/v*/w/_zaf.gif
 ||api.eigene.io/rec/
+||guj.de/js/gujTracker.js
+/tracking.js$domain=schoener-wohnen.de
```

Comment:
I don't see these requests. Do you still see them?

Reply:
@Alex-302 Yes.
![image](https://user-images.githubusercontent.com/76908410/113857197-ca6e0180-97d4-11eb-9444-1a1639219d95.png)

Reply:
Ok.